governance/working-group-chair-proposal-process.md

```diff
@@ -0,0 +1,82 @@
+Finally, they provide regular updates to the TAG, keeping the TAG leadership team and members informed about the group's progress and deliverables.
+
+## Caveats of Working Group Chairs
+
+* The position of a working group chair is not a paid position (TAG co-chairs and tech leads are also not paid positions)
+* Working group Chairs [have responsibilities](roles.md) to the Environmental Sustainability TAG, the leadership team, and the TOC
+* Nominations by the community, with endorsements by 1 or more working group/TAG co-chairs
+
+## Process of nominations
+
+Depending on the needs of the community, as determined by the Environmental Sustainability TAG leadership team, working group chair nominations may come in the form of "Nominees by community", "Nominees by working group co-chairs", "Nominees by TAG co-chairs" or all of them.
+
+The Environmental Sustainability TAG does not have a current limit on the amount of working group chairs.
+
+Final working group chair nominations will adhere to the [CNCF's TOC described process](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#elections).
+
+Nominees should adhere to the working group chair nomination requirements described below.
+Any individual who is a member of the Environmental Sustainability TAG list or the tag-environmental-sustainability slack channel may nominate another individual.
+
+The working group co-chairs reserve the right to endorse a nominee by the community based on:
+
+* Gender diversity
+* Company diversity
+* Geo diversity
+* Amount of nominations from different companies, people, regions for a single individual
+* Existing body of work
+* Justification provided during nomination
+
+#### Community timeline
+
+Community nominations will run for at least 1 month with a maximum up to 2 months.
+Upon close of nominations a decision will be made by the working group co-chairs within the next month.
+
+#### Submitting a nomination
+
+Nominations are to be submitted to [Environmental Sustainability TAG Co-Chairs email alias](mailto:cncf-tag-env-sus-chairs@lists.cncf.io) with the details:
+
+* Subject "Working Group $NAME-OF-WORKING-GROUP chair nomination: $NAME-OF-NOMINEE"
+* CC the nominee
+* Justification
+
+#### Endorsement of community nominees
+
+Working group co-chairs are responsible for providing endorsement of a community nominee to show concurrence and support.
+
+If no nominees are submitted by the community or from working group/TAG co-chairs, either as a result of lack of participation or lack of requirements, the working group co-chairs must endeavor to nominate an individual(s) that meet as many of the requirements as exist.
```

Comment:
or perhaps consider the activity of the group and whether it needs to continue.  If there are no individuals interested in leading, not much work can be achieved.

Reply:
@TheFoxAtWork @raravena80 thanks for the feedback! I wasn't able to find any official process for working group or TAG archival so I created a separate file in `governance` folder that outlines a baseline description of working group archival process.
I have linked it in the working group `charter.md`, as well as in the **Process of nominations** section of `working-group-chair-proposal-process.md`, instead of having an endorsement section. Please let me know if you disagree or if anything should be updated.
@leonardpahlke @nikimanoledaki FYI.